packages/cms-cli/commands/functions/build.js

```diff
@@ -0,0 +1,73 @@
+const ora = require('ora');
+const {
+  addPortalOptions,
+  addConfigOptions,
+  setLogLevel,
+  getPortalId,
+  addUseEnvironmentOptions,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cms-lib');
+const { logger } = require('@hubspot/cms-lib/logger');
+const { buildPackage } = require('@hubspot/cms-lib/api/functions');
+const { validatePortal } = require('../../lib/validation');
+
+const makeSpinner = (functionPath, portalIdentifier) => {
+  return ora(
+    `Building new bundle for '${functionPath}' on portal '${portalIdentifier}'.\n`
+  );
+};
+
+const loadAndValidateOptions = async options => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath, options);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validatePortal(options)))) {
+    process.exit(1);
+  }
+};
+
+exports.command = 'build <path>';
+exports.describe =
+  'builds a new dependency bundle for the specified function folder';
+
+exports.handler = async options => {
+  loadAndValidateOptions(options);
+
+  const { path: functionPath } = options;
+  const portalId = getPortalId(options);
+  const spinner = makeSpinner(functionPath, portalId);
+
+  trackCommandUsage('functions-build', { functionPath }, portalId);
+
+  logger.debug(`Starting build for function(s) with path: ${functionPath}`);
+
+  spinner.start();
+  await buildPackage(portalId, `${functionPath}/package.json`);
+  spinner.stop();
+  logger.success(
+    `Successfully built bundle from package.json for ${functionPath} on portal ${portalId}.`
+  );
+};
+
+exports.builder = yargs => {
+  yargs.positional('path', {
+    describe: 'Path to serverless function folder',
+    type: 'string',
+  });
+  yargs.example([['$0 functions build myFunctionFolder.functions']]);
```

Comment:
I don't think you need the inner array unless you want to add a description to the example, though that could be a good addition.

`Package/Build a deployable bundle for myFunctionFolder.functions`

Reply:
It looks like it has to be an array of arrays. It breaks if I remove the nested array:
![image](https://user-images.githubusercontent.com/6472448/97742266-6a9f5c00-1aba-11eb-9ce8-d885b1ce01d0.png)

Reply:
Hm ok my bad.  :) Still do we should add a description?

Reply:
Ah gotcha, added.